crates/sui-single-node-benchmark/src/command.rs

```diff
@@ -68,13 +69,16 @@ pub enum Command {
     },
 }
 
-#[derive(Copy, Clone, ValueEnum)]
+#[derive(Copy, Clone, EnumIter, ValueEnum)]
 pub enum Component {
     /// Baseline includes the execution and storage layer only.
     Baseline,
     /// On top of Baseline, this schedules transactions through the transaction manager.
     WithTxManager,
     /// This goes through the `handle_certificate` entry point on authority_server, which includes
     /// certificate verification, transaction manager, as well as a noop consensus layer.
-    ValidatorService,
+    ValidatorWithNoopConsensus,
+    /// Similar to ValidatorWithNoopConsensus, but the consensus layer would verify the transaction
+    /// and store the sequenced transactions into the store.
+    ValidatorWithDirectConsensus,
```

Comment:
I think we can mention this only runs part of consensus handler, or name this `ValidatorWithFakeConsensus`.